services/src/pro/users/postgres_userdb.rs

```diff
@@ -66,17 +66,23 @@ where
 
         let stmt = tx
             .prepare(
-                "INSERT INTO users (id, email, password_hash, real_name, active) VALUES ($1, $2, $3, $4, $5);",
+                "INSERT INTO users (id, email, password_hash, real_name, quota_available, active) VALUES ($1, $2, $3, $4, $5, $6);",
```

Comment:
If there is a quote no matter what is specified if it is used, shouldn't we set it explicitly to 0 in the settings. Currently, it is commented out.

Reply:
Right now, if no quota is set, it would just default to zero. If we make it non-optional and no quota is set, geo engine will crash at runtime instead, when the first workflow is run. If the latter is our preferred behavior, we can make it non-optional. I thought taking 0 made sense.